src/WebHandlerImpl.h

```diff
@@ -83,7 +83,13 @@ class AsyncCallbackWebHandler: public AsyncWebHandler {
       if(!(_method & request->method()))
         return false;
 
-      if(_uri.length() && (_uri != request->url() && !request->url().startsWith(_uri+"/")))
+      if (_uri.length() && _uri.endsWith("*")) {
+        String uriTemplate = String(_uri);
+        uriTemplate.replace("*", "");
```

Comment:
Use `String::substring()` instead of replace

Reply:
it's the last symbol so it is easy :)

Reply:
Ok. Thanks for the remark!